src/pages/saksoversikt/simulering/simulering.tsx

```diff
@@ -40,12 +47,23 @@ const Utbetalingssimulering = (props: { utbetaling: Utbetaling }) => {
                         </tr>
                     </thead>
                     <tbody>
-                        {props.utbetaling.simulering.perioder.map((periode, index) => (
-                            <tr key={index}>
-                                <td>{`${intl.formatDate(periode.fom)} - ${intl.formatDate(periode.tom)}`}</td>
-                                <td>{periode.bruttoYtelse}</td>
-                            </tr>
-                        ))}
+                        {gruppertSimuleringsperioder.map((gruppe, index) => {
+                            return pipe(
+                                combineOptions(arr.head(gruppe), arr.last(gruppe)),
+                                Option.fold(
+                                    () => null,
+                                    ([head, last]) => (
+                                        <tr key={index}>
```

Comment:
Hva med f.eks.
```suggestion
                                        <tr key={head.fom+last.tom}>
```
?

Reply:
Ordningen på simuleringsperioderna kan inte ändar sig. Men ja, kan ha det sån så att man slipper ha in en extra parameter! 😊